<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceMeasurementTool</class>
 <widget class="qMRMLWidget" name="SurfaceMeasurementTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>311</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLNodeComboBox" name="surfaceSelector">
     <property name="nodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLModelNode</string>
      </stringlist>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="computeButton">
     <property name="text">
      <string>Compute Measurements</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="areaLabel">
     <property name="text">
      <string>Area: -</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bboxLabel">
     <property name="text">
      <string>Bounding box: -</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="centerLabel">
     <property name="text">
      <string>Center of mass: -</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SurfaceMeasurementTool</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>surfaceSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
